bootstrap/helpers.php

```diff
@@ -1,6 +1,6 @@
 <?php declare(strict_types = 1);
 
-use DealerLive\Config\Services\Configuration;
+use Core\Services\Configuration;
```

Comment:
You can't reference an external class in a support method. While it may have done it before, that should exist here.
Move this method into a helper.php file present in the glovebox project, and delete this one.

Reply:
Deleted this function and moved it to a helper.php in Glovebox